lib/abide_dev_utils/xccdf.rb

```diff
@@ -218,6 +229,19 @@ def controls_by_profile_title(profile_title)
         profiles.select { |x| x.title == profile_title }.controls
       end
 
+      def gen_map(opts)
+        os, ver = facter_platform
+        if opts[:dir]
+          mapping_dir = File.expand_path(File.join(opts[:dir], opts[:type], os, ver))
+        else
+          mapping_dir = ''
+        end
+        ['title', 'hiera_title', 'hiera_title_num', 'number'].each_with_object({}) do |idx, h|
+          map_file_path = "#{mapping_dir}/#{idx}.yaml"
```

Comment:
If `mapping_dir` resolves to `''` in the case of `opts[:dir]` not being found, wouldn't `map_file_path` resolve to `/#{idx}.yaml`? I'm not sure we want it to default to trying to save files in `/`.

Reply:
In that case it's just using the base filename is the key in the hash.  mapping_dir isn't passed to AbideDevUtils::Output.yaml